ImageItem/ImageItem.module.less

```diff
@@ -0,0 +1,30 @@
+// ImageItem.module.less
+//
+
+@import '../styles/skin.less';
+@import '../styles/mixins.less';
+
+.imageItem {
+  	.applySkins({
+		border: @agate-imageitem-border-width solid @agate-imageitem-border-color;
+
+		.focus({
+			&::before {
+				border-color: @agate-imageitem-focus-border-color;
+				background-color: @agate-imageitem-focus-bg-color;
+			}
+		});
+
+		.caption {
+	  		height: @agate-imageitem-caption-height;
+	  		line-height: @agate-imageitem-caption-height;
+	  		text-align: center;
+		}
+
+		&.horizontal {
+	  		.image {
+				margin: auto 0; // leaving this to align image with caption when orientation="horizontal"
+	  		}
+		}
```

Comment:
I don't think that we need to define `&.horizontal {` in `.applySkins({ ... })`. Could you pull it out of the `.applySkins({ ... })`?